fog/ledger/server/src/key_image_service.rs

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2018-2022 The MobileCoin Foundation
-use crate::{DbPollSharedState, SVC_COUNTERS};
+use crate::{metrics::*, DbPollSharedState, SVC_COUNTERS};
```

Comment:
We generally avoid `*` imports, you can import `metrics` and then use `metrics::` everywhere or import the metrics directly (or we can decide we're fine with `*` imports)